docs/root/api-v3/config/config.rst

```diff
@@ -17,3 +17,4 @@ Extensions
   grpc_credential/grpc_credential
   retry/retry
   trace/trace
+  compression/decompressor
```

Comment:
This line (together with the `decompressor.rst` file) is not really needed, since `compression/compression.rst` takes care of including compression-related API docs already.